Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 COPY Gemfile Gemfile.lock /usr/src/app/
-RUN bundle install --without test production --jobs 2
```

Comment:
👍

Reply:
Still need to bundle install something, maybe just remove `production` from that line